### ISSUE TITLE
Step 025: Application::Terminate() is never called

### ISSUE THREAD

**Issue by rojsc** [eliemichel/LearnWebGPU-Code#45]
`Application::Terminate()` is defined and described in the tutorial, but it seems to be never called in the `step025` branch.

**Comment by eliemichel** [comment]
Indeed, and the problem comes from step20 actually, thanks for noticing this!
